src/app/features/exchange-set/ess-list-encs/ess-list-encs.component.ts

```diff
@@ -117,4 +130,24 @@ export class EssListEncsComponent implements OnInit {
   displaySingleEnc() {
     this.displaySingleEncVal = true;
   }
+
+  requestEncClicked()
+  {
+    this.displayLoader = true;
+    this.msalService.instance.acquireTokenSilent(this.essSilentTokenRequest).then(response => {
```

Comment:
Just a general comment - no changes needed

In future I think it would be cleaner to move the token acquiring code into the service classes. It seems to me it should not be a concern of the UI components

Reply:
Noted. Thanks